docs/learn/benefits-lukso-standards.md

```diff
@@ -34,7 +26,18 @@ The table below highlights the features of the LSPs and their benefits:
 | Upgradeable Security           | ✅                         | ✅                          | ✅                                      | ✅ Through [LSP14](../standards/access-control/lsp14-ownable-2-step.md)                             |
 | Pre and Post Execution Hooks   | ✅                         | ✅                          | ✅                                      | ✅ Through [LSP20](../standards/accounts/lsp20-call-verification.md)                                |
 
-As shown from the table above, the [Universal Profile Browser Extension](/install-up-browser-extension) provides more features and a better user experience when interacting with dApps. Universal Profiles (known as 🆙) are smart contract accounts that offer many benefits.
+| Feature                                        | Benefits                                                                                                                                                                                                                                                                                                                           |
+| ---------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **🗄️ Metadata in one place**                   | Finished re-filling the same information again and again on each new app you register to!<br/>Your information is stored and fetched from the same place (your 🆙 storage) and displayed the same across all the dApps ecosystem!                                                                                                  |
```

Comment:
Hmm, "finished re-filling" sounds a little bit strange. `No more re-filling` sounds better but with that wording we will have 2 sentences starting identically. 
```suggestion
| **🗄️ Metadata in one place**                   | Removes the requirement to refill the same information again and again on each new app you register to!<br/>Your information is stored and fetched from the same place (your 🆙 storage) and displayed the same across all the dApps ecosystem!                                                                                                  |
```